Source/WebKit/UIProcess/API/Cocoa/_WKWebExtension.mm

```diff
@@ -275,6 +275,11 @@ - (BOOL)hasCommands
     return _webExtension->hasCommands();
 }
 
+- (BOOL)hasDeclarativeNetRequestRules
+{
+    return !_webExtension->declarativeNetRequestRulesets().isEmpty();
```

Comment:
This would be best to have a method on `WebKit::WebExtension` to call. While this is simple now, it might get more complex in the future and these API wrappers should ideally have almost no logic.

It can just be an inline function in WebExtension.h that does `isEmpty()` for now.

Reply:
Got it! So then the extra logic will go into WebExtensionCocoa.mm instead of _WKWebExtension.